CMakeLists.txt

```diff
@@ -13,8 +13,10 @@ option(BUILD_UNIVERSAL "build both arm64 and x86_64 on macOS" OFF)
 option(INSTALL_PYTHON "install python ffi" OFF)
 option(INSTALL_RESOURCE "install resource" OFF)
 option(INSTALL_DEVEL "install development files" OFF)
+option(INSTALL_FLATTEN "do not use bin lib include directory" ON)
 option(INSTALL_THIRD_LIBS "install third party libraries" ON)
 option(USE_MAADEPS "use third-party libraries built by MaaDeps" ON)
+option(WITH_EMULATOR_EXTRAS "build with emulator extras" ON)
```

Comment:
It's important for us: https://github.com/NixOS/nixpkgs/pull/311279#issuecomment-2128302129

good job

Reply:
There's no such thing like Mumu on Linux and MacOS.

```
IF (NOT ${CMAKE_SYSTEM_NAME} MATCHES "Windows")
    SET(WITH_EMULATOR_EXTRAS OFF)
ENDIF()
```